documentation/content-standards.md

```diff
@@ -47,7 +47,9 @@ We'll describe the standards for each of these components separately.
 All entry file names use the `.md` format along with the following standards:
 
 - They must match the title of the entry. (e.g., "Three.js" -> `three-js.md`)
-- It must be separated by dashes, where appropriate. (e.g., "Data Types" -> `data-types.md`, "HashTable" -> `hashtable.md`)
+- It must be separated by dashes, where appropriate. (e.g., "Data Types" -> `data-types.md`)
+  - If the title or name is meant to be in PascalCase: "HashTable" -> `hashtable.md`
+  - If the title or name is meant to be in camelCase: ".forEach()" -> `forEach.md`.
 - The name should always be lowercase. (e.g., "Array" -> `array.md`, "HTML" -> `html.md`)
```

Comment:
As its currently phrased, I'm not sure how this squares up with the list items above. Maybe we could rephrase as something like this:

```suggestion
- The name should always begin with a lowercase letter. (e.g., "Array" -> `array.md`, "HTML" -> `html.md`)
```